docs/src/prototypes/customToolbar/darkThemeOverrides.ts

```diff
@@ -0,0 +1,227 @@
+  ctItemDangerBackgroundHover: string
+
+  ctItemIndicatorPadding: string
+
+  ctItemNotificationBackgroundColor: string
+  ctItemNotificationSize: string
+
+  ctItemPrimaryBackground: string
+  ctItemPrimaryBackgroundHover: string
+  ctItemPrimaryColorHover: string
+}
+
+export const darkThemeOverrides: ThemeInput = {
+  componentVariables: {
+    Toolbar: (siteVars): CustomToolbarVariables => ({
+      ctBorderRadius: '4px',
+      ctBorderStyle: 'solid',
+      ctBorderWidth: '2px',
+      ctHeight: '4rem',
+
+      ctItemBackground: siteVars.colorScheme.default.background1,
+      ctItemBackgroundHover: siteVars.colorScheme.brand.backgroundHover1,
+      ctItemBorderColorFocus: siteVars.colorScheme.default.borderFocus,
+      ctItemColor: siteVars.colorScheme.default.foreground,
+      ctItemColorFocus: siteVars.colorScheme.default.foregroundFocus,
+      ctItemColorHover: siteVars.colorScheme.default.foregroundHover,
+
+      ctItemActiveBackground: siteVars.colorScheme.brand.backgroundHover1,
+      ctItemActiveBackgroundOverlay:
+        'linear-gradient(90deg,rgba(60,62,93,.6),rgba(60,62,93,0) 33%),linear-gradient(135deg,rgba(60,62,93,.6) 33%,rgba(60,62,93,0) 70%),linear-gradient(180deg,rgba(60,62,93,.6) 70%,rgba(60,62,93,0) 94%),linear-gradient(225deg,rgba(60,62,93,.6) 33%,rgba(60,62,93,0) 73%),linear-gradient(270deg,rgba(60,62,93,.6),rgba(60,62,93,0) 33%),linear-gradient(0deg,rgba(98,100,167,.75) 6%,rgba(98,100,167,0) 70%)',
+      ctItemActiveColor: siteVars.colorScheme.default.foreground3,
+
+      ctItemDangerBackground: siteVars.colorScheme.red.background2,
+      ctItemDangerBackgroundHover: siteVars.colorScheme.red.backgroundHover,
+      ctItemDangerColorHover: siteVars.colorScheme.red.foregroundHover,
+
+      ctItemIndicatorPadding: '8px',
+
+      ctItemNotificationBackgroundColor: 'red',
+      ctItemNotificationSize: '8px',
+
+      ctItemPrimaryBackground: siteVars.colorScheme.default.background3,
+      ctItemPrimaryBackgroundHover: siteVars.colorScheme.brand.backgroundHover1,
+      ctItemPrimaryColorHover: siteVars.colorScheme.brand.foregroundHover1,
+    }),
+  },
+
+  componentStyles: {
+    Toolbar: {
+      root: ({
+        variables: v,
+      }: ComponentStyleFunctionParam<ToolbarProps, CustomToolbarVariables>) => ({
+        ...(v.isCt && {
+          borderRadius: v.ctBorderRadius,
+          height: v.ctHeight,
+          overflow: 'hidden',
+        }),
+      }),
+    },
+
+    ToolbarCustomItem: {
+      root: ({
+        props: p,
+        variables: v,
+      }: ComponentStyleFunctionParam<ToolbarCustomItemProps, CustomToolbarVariables>) => ({
+        ...(v.isCt && {
+          background: v.ctItemBackground,
+          borderStyle: v.ctBorderStyle,
+          borderWidth: v.ctBorderWidth,
+
+          ...(v.isCtItemPrimary && { background: v.ctItemPrimaryBackground }),
+          ...(v.isCtItemIndicator && { padding: v.ctItemIndicatorPadding }),
+
+          ...(p.isFromKeyboard && {
```

Comment:
Let's check with design whether these focusing styles are correct. And at least add comment that those should be changed if they should appear as in the other components.

Reply:
/cc @codepretty

Reply:
This rather shows how to change styles for focus than matches redlines.

Reply:
I agree, just if we can use the general `getBorderFocusStyles` styles, we would be able to remove some styles... Let's not address this now until we are sure what is the expected state here...